### PR TITLE
Integrate help pane into output area under game screen

### LINKED ISSUE #1109: Make tutorial pane resizable
the tutorial pane should be resizable (like the game window is) so that it can be expanded for easier reading when needed. this is especially important for tutorials/jams where you need to read through lots of code or text.

### PR DESCRIPTION
**Pull request by polypixeldev**
This PR moves the help and tutorial pane to be under the game screen instead of in a popup pane. This makes it easier to be coding, testing, and reading without having to be hiding and showing the help pane.
It also makes it resizable with a resize bar which is linked to the vertical resize bar to ensure that the game screen and help pane don't overlap.
Finally, it saves the scroll location of both the tutorial and toolkit tabs in state so that you can switch between them without having to scroll again.

Still needs to be tested on a variety of browsers and screen sizes to make sure it doesn't break.

Fixes #1109

### CODE REVIEW

**Comment by V205Arduino** [comment]
Currently testing.
How do I resize the tutorial window thingy?
New Note

**Comment by polypixeldev** [comment]
You should be able to use the vertical resize bar to the left or the horizontal resize bar above the tutorial panel

**Comment by V205Arduino** [comment]
Is it supposed to be like this:
New Note
Maybe the space could be decreased?
It is a bit far.....
New Note
Should the circled space be decreased?

**Comment by polypixeldev** [comment]
Yeah, it should. Was having a little trouble getting it to work, CSS is weird - I'll take a look tomorrow

**Comment by grymmy** [comment]
@polypixeldev please assign to me when you are ready for this to be reviewed

**Comment by JosiasAurel** [comment]
@polypixeldev please resolve conflicts so we can move ahead

**Comment by JosiasAurel** [comment]
should also resolve #829

**Comment by polypixeldev** [comment]
Decided to just remove the horizontal help area resize bar. Since the game view has defined proportions, a horizontal resize bar would have to adjust the vertical one as well, so it's redundant. Makes it so that the issue with too much space between game and help is gone.

**Comment by grymmy** [comment]
This looks great!

File: src/components/big-interactive-pages/editor.tsx [hunk 1]
```diff
@@ -33,7 +34,9 @@ interface ResizeState {
 
 const minOutputAreaSize = 360
 const defaultOutputAreaSize = 400
-const heightMargin = 130
+const widthMargin = 130
+
+const minHelpAreaSize = 200
```
Comment (grymmy) [inline src/components/big-interactive-pages/editor.tsx:39]: nit - Suggest adding comments to all of these constants describing exactly what they represent (minHelpAreaSize - is it width or height, for instance)

**Review by grymmy — APPROVED** [review]
Left one nit but in general looks fine to land

**Pull request merged**